projects/cashmere/src/lib/chip/chip-row/chip-row.component.ts

```diff
@@ -10,10 +11,11 @@ import {parseBooleanAttribute} from '../../util';
     encapsulation: ViewEncapsulation.None
 })
 export class ChipRowComponent {
-    _wrap: boolean = true;
+    private _wrap: boolean = true;
```

Comment:
this needs to be public since it's used in the template

Reply:
Leave the private off. We should be following the standard of leaving the public modifier off since that is the default access level